lib/ruby-enum/enum.rb

```diff
@@ -118,7 +118,13 @@ def keys
 
       # Returns all enum values.
       def values
-        @_enum_hash.values.map(&:value)
+        values = @_enum_hash.values.map(&:value)
```

Comment:
We are shadowing `values` with a local `values` here, maybe rename the local one to `result` or `all` or something better?

Reply:
Fixed.